File: twa/src/main/java/com/webbrowser/MainActivity.java
package com.webbrowser;

import android.content.Intent;
import android.net.Uri;
import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import com.google.androidbrowserhelper.trusted.LauncherActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
    /*    TwaLauncher launcher = new TwaLauncher(this);
        launcher.launch(Uri.parse("https://google.com"));*/
        Intent intent = new Intent(this, LauncherActivity.class);
        intent.setData(
                Uri.parse("http://twitch.com/"));
        startActivity(intent);
        finish();
    }
}

File: twa/src/main/java/com/webbrowser/MainActivity.kt
package com.webbrowser

import android.app.Activity
import android.content.Intent
import android.net.Uri
import android.os.Bundle
import com.google.androidbrowserhelper.trusted.LauncherActivity

class MainActivity : Activity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val intent = Intent(this, LauncherActivity::class.java)
//        intent.setData(Uri.parse("http://192.168.1.142:8000"))
        intent.setData(Uri.parse("https://www.superlist.com"))
        startActivity(intent)
        finish()
    }
}